client/packages/common/src/intl/locales/en/common.json

```diff
@@ -354,6 +355,7 @@
   "log.changed-to": "to",
   "log.invoice-created": "Shipment created",
   "log.invoice-deleted": "Shipment deleted",
+  "log.zero-quantities-for-invoice-created-before-store": "The quantity of all lines has been set to 0 and invoice has been finalised as it was created before the store.",
```

Comment:
Not too sure about this message... open for suggestions. Didn't wanna make it too long 👀